src/lib/components/forms/UserFeedback.svelte

```diff
@@ -0,0 +1,218 @@
+<script lang="ts">
+  import { createEventDispatcher } from "svelte";
+  import { _ } from "svelte-i18n";
+  import { Bug16, Comment16, Question16 } from "svelte-octicons";
+  import type { User } from "@/api/types";
+  import { createFeedback, type Feedback } from "@/lib/api/feedback";
+  import Button from "../common/Button.svelte";
+  import Flex from "../common/Flex.svelte";
+  import UserAvatar from "../accounts/UserAvatar.svelte";
+  import { toast } from "../layouts/Toaster.svelte";
+
+  export let user: User = undefined;
+
+  let feedback = "";
+  let feedbackType = "Comment";
+
+  let feedbackTypes = [
+    {
+      id: "comment",
+      value: "Comment",
+      label: $_("feedback.types.comment.label"),
+      placeholder: $_("feedback.types.comment.placeholder"),
+      icon: Comment16,
+    },
+    {
+      id: "bug",
+      value: "Bug",
+      label: $_("feedback.types.bug.label"),
+      placeholder: $_("feedback.types.bug.placeholder"),
+      icon: Bug16,
+    },
+    {
+      id: "question",
+      value: "Question",
+      label: $_("feedback.types.question.label"),
+      placeholder: $_("feedback.types.question.placeholder"),
+      icon: Question16,
+    },
+  ];
+
+  let anonymous = !Boolean(user);
+
+  const dispatch = createEventDispatcher();
+  $: placeholder =
+    feedbackTypes.find((type) => type.value === feedbackType)?.placeholder ??
+    $_("feedback.defaultPlaceholder");
+
+  let status: null | "loading" | "success" | "error" = null;
+
+  // For simplicity, we directly call the Baserow API from the client.
+  // Unfortunately, this exposes our API token and endpoint to end users.
+  // If we want a more secure approach, we can provide the submission logic
+  // in SvelteKit server logic, though that approach is more complex.
+  async function handleSubmit(e: SubmitEvent) {
+    status = "loading";
+    const form = e.target as HTMLFormElement;
+    const fd = new FormData(form);
+
+    // POST form data to baserow
+    const data: Feedback = {
+      Type: String(fd.get("type")) ?? "",
+      Message: String(fd.get("message")) ?? "",
+      User: String(fd.get("user")) ?? "",
+    };
+    try {
+      await createFeedback(data);
+      status = "success";
+      toast($_("feedback.success"), {
+        status: "success",
+      });
+      dispatch("close");
+    } catch (e) {
+      toast(String(e), { status: "error" });
+      status = "error";
+    }
+  }
```

Comment:
@eyeseast Since you have more experience so far working with SvelteKit server actions, I leave this to your discretion.